docs/devices/E1603_E1702.md

```diff
@@ -61,8 +61,8 @@ This device supports OTA updates, for more information see [OTA updates](../info
 ## Exposes
 
 ### Switch 
-The current state of this switch is in the published state under the `state` property (value is `ON` or `OFF`).
-To control this switch publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/set` with payload `{"state": "ON"}`, `{"state": "OFF"}` or `{"state": "TOGGLE"}`.
+The current state of this switch is in the published state under the `state` property (value is `ON` or `OFF`). To control this switch publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/set` with payload `{"state": "ON"}`, `{"state": "OFF"}` or `{"state": "TOGGLE"}`.  
+This switch also supports the "On With Timed Off" command. It allows the switch to turn on for a specified duration and then turn off automatically. For example to turn on for five minutes, use the payload `{"state": "onWithTimedOff", "on_time": 300}`. Durations are specified in seconds and fractions are supported at up to 1/10th of a seconds. See chapter 3.8.2.3.6 of the ZCL for more advanced scenarios.  
```

Comment:
Shouldn't `{"state": "onWithTimedOff", "on_time": 300}` be `{"state": "on", "on_time": 300}` ?

Reply:
Oops, true!